x-pack/plugins/apm/e2e/cypress/integration/csm_dashboard.feature

```diff
@@ -30,11 +25,6 @@ Feature: CSM Dashboard
     Then should display percentile for page load chart
       And should display tooltip on hover
 
-  Scenario: Breakdown filter
```

Comment:
Are we able to comment these out instead with a comment stating they are flaky so that the Uptime team can later review? I am unfamiliar with the `.feature` file type, so I genuinely don't know.

Reply:
me neither 😅, that's why I deleted it. But it could be restored on GitHub anyway history.